client/src/components/pages/LoginPage/style.css

```diff
@@ -0,0 +1,7 @@
+.login-page--main-section {
+    min-height: calc(100vh - calc(129px + 70px));
```

Comment:
@ahmedisam99 Is this calc intended to make the login section height equal to all of the page minus the header's height?
If so, it might be helpful to have a `.page` container div around the header and login section and make it flex (column, height: 100vh), so that the section underneath the header always takes up all of the remaining page height.  You could put these styles into App.css so that you could make every page have the same dom structure and automatically get the same vertical sizing behavior.
That might be better since you could change the header height and all of the pages would still have the correct layout, without having to update a bunch of individual page css files.  And it would be a little less effort each time you need to create / style a new page.

Reply:
After all of that, you could also refactor even further and move the "same dom structure" that I mentioned into a HOC, for example `withHeader`, which takes your page component and wraps it in a page container and adds a header above it.

But all of these are just suggestions and ideas, so no need to make a bunch of changes to your PR :)

Reply:
Nevermind, the changes I described already happenened in #51 :)